e2e/Utils.ts

```diff
@@ -260,3 +261,7 @@ export const normalizeIcons = (str: string) => {
     .replace(new RegExp('\u00D7', 'g'), '\u2715')
     .replace(new RegExp('\u221A', 'g'), '\u2713');
 };
+
+// Certain environments (like CITGM and GH Actions) do not come with mercurial installed
+export const hgIsInstalled = which.sync('hg', {nothrow: true}) !== null;
+export const testIfHg = hgIsInstalled ? test : test.skip;
```

Comment:
@SimenB By moving the code in `Utils` - _which might be called by others_ - don't we have the issue that the call to `which.sync` will be done for files that do not need it?

I was thinking to update it that way:
```suggestion
// Certain environments (like CITGM and GH Actions) do not come with mercurial installed
export const hgIsInstalled = () => which.sync('hg', {nothrow: true}) !== null;
```
And let the caller do the: `const testIfHg = hgIsInstalled() ? test : test.skip`.
What do you think?

Reply:
```ts
export const testIfHg = () => (which.sync('hg', {nothrow: true}) !== null) ? test : test.skip;
```

?

Reply:
It would mean that we re-do the call to which it time we declare a test requiring testIfHg 🤔

Reply:
```ts
let hasHg = null;

export const testIfHg = () => {
  if (hasHg === null) {
    hasHg = which.sync('hg', {nothrow: true}) !== null;
  }

  return hasHg ? test : test.skip;
};
```
? 😀

Reply:
Looks good to me. I'll update my PR accordingly. Thanks

Reply:
I put a yet another suggestion. Actually it looked a bit strange IMO to do:
```js
testIfHg()("label", () => {})
// while normal tests are:
test("label", () => {})
```

I suggested an approach to keep the syntax:
```js
testIfHg("label", () => {})
```